CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Added
 
+- Added `Py42ActiveLegalHoldError` exception when attempting to deactivate a user or device in an active legal hold.
```

Comment:
Not a big deal here because it is obvious, but for SDK type projects, it is better to mention the exact methods being affected. In this case it would be `sdk.users.deactivate()` and `sdk.devices.deactivate()`.  Just 1 less mental connection we have to put on CHANGELOG consumers.